README.md

```diff
@@ -21,6 +25,40 @@ To test the complete result from a nix-build:
     $ nix-build
     $ nix-shell --run 'cd result/ && python -m http.server'
 
+### Docker Desktop for Mac
+
+Unfortunately it will not build on macOS natively. 
+But it can be build in a [Docker container on macOS](https://www.docker.com/products/docker-desktop). 
+(You may need to increase the maximum "Disk image size" in Docker's Resources Preferences.)
+
+In your favourite macOS terminal:
+
+    $ docker run -p 8000:8000 -it --name nixos-homepage nixos/nix
+
+Since git, fd and entr are not installed in the container:
+
+    # nix-shell -p git fd entr
+
+Following with the same steps as on Linux/NixOS:
+
+    [nix-shell]$ git clone https://github.com/NixOS/nixos-homepage.git
+    [nix-shell]$ cd nixos-homepage
+    [nix-shell]$ nix-shell
+    [nix-shell]$ make
+
+Add the `--bind 0.0.0.0` argument to bind the web server with the container's external interface too:
+
+    [nix-shell]$ python -m http.server --bind 0.0.0.0
```

Comment:
I think we could also use some other way to serve from the host environment (not from within docker) to serve a folder. Maybe simply running `nix-shell -p python3 --run "python -m http.server"` outside the container is better.

Reply:
@garbas, this is a great idea if we get the `make` issue fixed, as long as there is content dynamically generated in the backend. I experimented with your suggestion and I believe the interfaces ip address must be specified within the nix-shell, so: `nix-shell -p python3 --run "python -m http.server --bind 127.0.0.1"`(and the `-p 8000:8000` argument must be removed from the docker invocation).